Sources/App/Core/SearchFilter.swift

```diff
@@ -250,16 +292,27 @@ struct LastCommitSearchFilter: SearchFilter {
         return formatter
     }()
     
+    static var viewDateFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "d MMM yyyy"
+        formatter.locale = Locale(identifier: "en_US_POSIX")
+        formatter.timeZone = TimeZone(secondsFromGMT: 0)
+        
```

Comment:
We don't have one for dates but I wonder if we should pull this out so our formatters are a little more organised? Equally, I could do this as part of expanding the filters to include `last_active`. Noted for inclusion in #1390.

Reply:
Probably makes sense to have a generic "search filter date formatter" for all filters which have a date input. My typical coding brain would say refactor that once there's a need, so in 1390, though. I'm happy to sort out that rebase+cleanup if need be once this gets merged through.